### DISCUSSION TITLE
No commands

### DISCUSSION THREAD

**Discussion by Qamadrya** [TownyAdvanced/TownyResources#89] [Q&A] [answered]
I have the newest **townyadvanced** with **townychat**. But when I start writing **/tra**, **/tr**, **/townyresources:tra**, **/townyresources:tr**, then only the nicknames of the players who are online are displayed. I tried both old versions **towny** and old versions **townyresources**. But nothing helped.
Help me please.

**Comment by LlmDl** [comment, marked as answer]
File a ticket and include your full server startup.
